### PR TITLE
fix some possible typos in helper script

### PR DESCRIPTION
**Pull request by wpumacay**
This issue is related to #182 . It also fixes a similar problem when pickling the low-dimensional data (it was pickling the whole demo instead)

### CODE REVIEW

File: tools/dataset_generator.py [hunk 1]
```diff
@@ -163,7 +163,7 @@ def save_demo(demo, example_path):
 
     # Save the low-dimension data
     with open(os.path.join(example_path, LOW_DIM_PICKLE), 'wb') as f:
-        pickle.dump(demo, f)
+        pickle.dump([obs.get_low_dim_data() for obs in demo], f)
```
Comment (stepjam) [inline tools/dataset_generator.py:166]: Thanks for the PR!
This line should be reverted though, as if you call obs.get_low_dim_data, it actually squashes all of it into one big bumpy array, which means we loose the semantics of what those values mean.
The existing code works fine, and it doesn't a tually pickle any images, as if you look above, you'll see that we set all of the image data to None.

File: tools/dataset_generator.py [hunk 2]
```diff
@@ -243,7 +243,7 @@ def run(i, lock, task_index, variation_count, results, file_lock, tasks):
 
         task_env = rlbench_env.get_task(t)
         task_env.set_variation(my_variation_count)
-        obs, descriptions = task_env.reset()
+        descriptions, _ = task_env.reset()
```
Comment (stepjam) [inline tools/dataset_generator.py:246]: Good catch! Thanks

**Review by stepjam — CHANGES_REQUESTED** [review]

File: tools/dataset_generator.py (hunk 1 above)
Comment (wpumacay) [reply tools/dataset_generator.py:166]: Oh ok, my bad :smile:. I thought it was pickling the whole demo. Didn't catch that None setting. Will revert it then :+1:

Comment (wpumacay) [reply tools/dataset_generator.py]: Alright, reverted it on bd2027a commit.

**Comment by stepjam** [comment]
Thanks for the PR!

**Pull request merged**